buildtools/cli.go

```diff
@@ -771,3 +801,50 @@ func pythonNativeCmd(cmdName string, rtDetails *coreConfig.ServerDetails, target
 		return errors.New(fmt.Sprintf("python project type: %s is currently not supported", projectType.String()))
 	}
 }
+
+func terraformCmd(c *cli.Context) error {
+	if show, err := cliutils.ShowCmdHelpIfNeeded(c, c.Args()); show || err != nil {
+		return err
+	}
+	configFilePath, orgArgs, err := getTerraformConfigAndArgs(c)
+	if err != nil {
+		return err
+	}
+	cmdName, filteredArgs := getCommandName(orgArgs)
+	switch cmdName {
+	// Aliases accepted by terraform.
+	case "publish", "p":
+		return terraformPublishCmd(configFilePath, filteredArgs, c)
+	default:
+		return errorutils.CheckError(errors.New("Terraform command:\"" + cmdName + "\" is not supported. " + cliutils.GetDocumentationMessage()))
+	}
+}
+
+func getTerraformConfigAndArgs(c *cli.Context) (configFilePath string, args []string, err error) {
+	if show, err := cliutils.ShowCmdHelpIfNeeded(c, c.Args()); show || err != nil {
```

Comment:
Notice that this check also exists inside the above ```terraformCmd``` function. Is it also needed here?